Tasks/UseRubyV1/useruby.ts

```diff
@@ -0,0 +1,33 @@
+import * as os from "os";
+import * as path from "path";
+import * as task from "azure-pipelines-task-lib/task";
+import { installRubyVersion } from "./installer";
+import * as proxyutil from "./proxyutil";
+
+async function run() {
+    try {
+        const version: string = task.getInput("version", false);
+        const architecture: string = task.getInput("architecture", false) || os.arch();
+
+        // Install tool
+        if (version) {
+            task.setResourcePath(path.join(__dirname, "task.json"));
```

Comment:
We should probably just set this regardless of whether we're installing the tool. It might not be completely necessary right now, but when we add things like auth we'll want localization for more than just installation (and its fast to do this).

Reply:
Moved up to start of script